node-db/lesson1.md

```diff
@@ -1,32 +1,673 @@
+
+server.listen(5000); 	
+
+console.log('Node.js web server at port 5000 is running..')
+```
+### Exercise
+Can you add another route `/mentor`?  
+
+# What is HTTP?
+
+> It is a protocol that browser and the server uses to talk to each other  
+
+![HTTP](https://github.com/Michael-Antczak/syllabus/blob/scotland/node-db/http_diagram.png)  
+
+Read more on Mozilla's [An overview of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
+
+
+# Node.js and Express
+
+[Express](http://expressjs.com/) is one of the most widely-used frameworks for Node.js. It simplifies base features of Node.js, making it easier and faster to build your application's backend. Learning Express gives you a great foundation for becoming a Node.js developer.
+
+In the next steps, we will building a **CMS (Content Management System)** - a system to write, publish and save blog posts. Even though we could use core libraries like `http` to build this system, this becomes very complicated and non-maintainable very quickly. That is why we use frameworks like `Express`.
+
+## Before you Start
+
+Fork the repository https://github.com/CodeYourFuture/express-workshop
+
+`$ git clone https://github.com/YOUR-USERNAME/express-workshop`
+
+# Step 1 - Setting up your project
+
+When creating a Node.js project, you will be installing a lot of different things along the way. If you want to share your project with others, you need to have a list of the things you installed, so that other people know what to install in order to run the project.
+
+In Node.js, this 'list' file is called a `package.json`. The 'things you've installed' are referred to as **dependencies**. Creating this file is the first step in setting up your Node.js project. 
+
+## 1. Make a `package.json` file
+
+Let's start by creating the `package.json` file. We can add things to it as the project grows. The `package.json` file is easy to create from the command line. Type the following command into your terminal to get started:
+
+```
+$ npm init
+```
+
+This command will initialise a step-by-step process for creating the `package.json`. You should see something like this:
+
+![starting npm init](https://raw.githubusercontent.com/node-girls/workshop-cms/master/readme-images/step1-npm-init01.png)
+
+It will ask you a bunch of questions.
+
+> You can skip most of the questions but change the `entry point` from `(index.js)` to `server.js`.
+
+> The wizard asks you for the following information: `name`, `version`, `description`, `main`, `test`, `repository`, `keywords`, `author`, `license` - do you understand all of them?
+
+At the endo of the wizard, you should see a new file called `package.json` in your project's folder.
+
+Here is an example `package.json` file for a project called [Passport](https://github.com/jaredhanson/passport/blob/master/package.json).
+
+# What is JSON?
+
+JSON is a type of file for structuring data in a readable way. It is also a really popular format for sending data across the web.
```

Comment:
I'd mention what the acronym stands for, JavaScript Object Notation.

Also I'd add...
```diff
-a readable way
+a human-readable way
```